### PR TITLE
[R4R] - {0.4.2}: fix qa deployer config

### PR DESCRIPTION
**Pull request by Sha3nS**
# Goals of PR

Core changes:
fix deployer config by main

Notes:
no

Related Issues:
no

### CODE REVIEW

File: packages/contracts/deploy-config/goerli-qa.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
   numDeployConfirmations: 1,
-  l1BlockTimeSeconds: 15,
-  l2BlockGasLimit: 15_000_000,
-  l2ChainId: 1705003,
-  updateGaslimitBlock: 90000,
-  eigenDaBlock: 2512521,
+  gasPrice: 150_000_000_000,
+  l1BlockTimeSeconds: 12,
+  l2BlockGasLimit: 30_000_000,
+  l2ChainId: 5000,
```
Comment (Tri-stone) [inline packages/contracts/deploy-config/goerli-qa.ts]: `L2ChainID` of QA network based on Goerli looks wrong.

File: packages/contracts/deploy-config/local.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-  l1BlockTimeSeconds: 15,
-  l2BlockGasLimit: 15_000_000,
-  l2ChainId: 17,
-  updateGaslimitBlock: 10,
-  eigenDaBlock: 100,
+  numDeployConfirmations: 1,
+  gasPrice: 150_000_000_000,
+  l1BlockTimeSeconds: 12,
+  l2BlockGasLimit: 30_000_000,
+  l2ChainId: 5000,
```
Comment (Tri-stone) [inline packages/contracts/deploy-config/local.ts]: `L2ChainID` of Local network looks wrong.

**Review by Tri-stone — APPROVED** [review]

File: packages/contracts/deploy-config/goerli-qa.ts (hunk 1 above)
Comment (Sha3nS) [reply packages/contracts/deploy-config/goerli-qa.ts]: done

File: packages/contracts/deploy-config/local.ts (hunk 2 above)
Comment (Sha3nS) [reply packages/contracts/deploy-config/local.ts]: done

**Review by wukongcheng — APPROVED** [review]

**Review by lqS0317 — APPROVED** [review]

**Pull request merged**